config/game_options.txt

```diff
@@ -398,13 +398,14 @@ JOIN_WITH_MUTANT_RACE
 ## You probably want humans on your space station, but technically speaking you can turn them off without any ill effect
 ROUNDSTART_RACES human
 
-## Races that are okay-ish on the station. Safe to enable.
-ROUNDSTART_RACES unathi
-#ROUNDSTART_RACES plasmaman
-#ROUNDSTART_RACES slime
-ROUNDSTART_RACES ethari
-#ROUNDSTART_RACES pod
-ROUNDSTART_RACES ipc
+## Races that are okay-ish on the station. Safe to enable. The number is the minimum hours required to play.
+ROUNDSTART_RACES unathi 10
+#ROUNDSTART_RACES plasmaman 0
+#ROUNDSTART_RACES slime 10
+ROUNDSTART_RACES ethari 0
```

Comment:
If you want to timelock either unathi or ethari - you pick ethari, every time.

Either don't timelock either, or do Ethari instead, yo.